File: examples/user/readSession/node/readASessionByTheSessionId.js
const { UserService } = require("m3o/user");

// Read a session by the session id. In the event it has expired or is not found and error is returned.
async function readAsessionByTheSessionId() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.readSession({
    sessionId: "df91a612-5b24-4634-99ff-240220ab8f55",
  });
  console.log(rsp);
}

readAsessionByTheSessionId();

File: examples/user/readSession/go/readASessionByTheSessionId.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Read a session by the session id. In the event it has expired or is not found and error is returned.
func ReadAsessionByTheSessionId() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.ReadSession(&user.ReadSessionRequest{
		SessionId: "df91a612-5b24-4634-99ff-240220ab8f55",
	})
	fmt.Println(rsp, err)
}
